server.py

```diff
@@ -38,6 +38,22 @@ def add_note():
 
     return jsonify(response)
 
+@app.route('/notes/edit/<id>', methods=['POST'])
+def update_edited_note_in_BD(id):
+    """ Replace Note in Database by ID with Edited Note and Title """
+
+    note_id = request.form.get('note_id')
+    note_title = request.form.get('note_title')
+    note_content = request.form.get('note_content')
+
+    
+    note = update_note(note_id, note_title, note_content)
+    print "update is running RUNRUNRUNRUN", note.content, note.created_at
+
+    flash("Your Edits Have Been Saved")
+
+    return redirect('/')
```

Comment:
Get rid of the `print`.

I only see this being called via AJAX.  What does `flash()` do in this case?  How is the 3xx response handled?  Does it think it's an error?  Does it follow the redirect and return the contents of `/` to your callbacks?